make-dist

```diff
@@ -168,6 +168,17 @@ echo "including src/.git_version, ceph.spec"
 
 (git rev-parse HEAD ; echo $version) 2> /dev/null > src/.git_version
 
+source /etc/os-release
```

Comment:
In `do_cmake.sh` (but not in `install-deps.sh`):
```suggestion
if [ -r /etc/os-release ]; then
    source /etc/os-release
```

Reply:
Good idea, will do.

Reply:
Done, re-tested, works fine.